NodeBase/Dockerfile

```diff
@@ -79,17 +79,13 @@ USER seluser
 #==============================
 # Scripts to run Selenium Node
 #==============================
-COPY entry_point.sh \
-  functions.sh \
-    /opt/bin/
-
+COPY entry_point.sh  /opt/bin/
```

Comment:
There is no need to update this file, please revert the changes ->
```
# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
# NOTE: DO *NOT* EDIT THIS FILE.  IT IS GENERATED.
# PLEASE UPDATE Dockerfile.txt INSTEAD OF THIS FILE
# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Reply:
Sorry, I don't know how I missed this!